### ISSUE TITLE
Check for disabled symlink function

### ISSUE THREAD

**Issue by fritzmg** [contao/contao-manager#146]
If the `symlink` function is disabled via `disable_functions` in the PHP settings, the Contao Install Tool will be unable to create the symlinks of course - but without telling you why. See https://github.com/contao/installation-bundle/issues/65

In general I think that the Contao Manager should also check for everything that the Contao Check already does for Contao 4 (see https://github.com/contao/check/blob/master/check/controller/contao4.php) - unless it already does that, I am not too familiar with the Contao Manager yet :)

**Comment by aschempp** [comment]
Great reference! Are there any other "default" checks run by the Contao Check or is that all that's run for Contao 4 ?

**Comment by fritzmg** [comment]
So basically, for Contao 4 these checks are made: [contao4.php#L50-L84](https://github.com/contao/check/blob/master/check/controller/contao4.php#L50-L84)
Plus for Contao 4.5+ this check: [contao45.php#L59-L61](https://github.com/contao/check/blob/master/check/controller/contao45.php#L59-L61)

To summarize:

* Minimum PHP version (for Contao 4.4 and 4.5+)
* Presence of at least GD, Imagick or Gmagick
* Presence of the PHP `intl` extension
* Ability to write the system tmp directory
* Ability to use the PHP `symlink()` function
* Ability to actually create symlinks
* Presence of the PHP `xmlreader` extension
* Ability to create file locks on existing files
* Minimum ICU version

**Comment by aschempp** [comment]
I've copied the checks to the Contao Manager, but the symlink check fails on Windows. Any idea why that could be?

**Comment by fritzmg** [comment]
Under Windows, the `SE_CREATE_SYMBOLIC_LINK_NAME` security policy must be enabled for the user. Alternatively, the web server process must be run with administrator rights.

**Comment by aschempp** [comment]
So this is nothing the manager can fix, but the admin has to setup?

**Comment by fritzmg** [comment]
I can have a look at it myself, just need an explanation on how to run and debug the Contao Manager without actually building the phar :) if that's possible?

**Comment by aschempp** [comment]
https://github.com/contao/contao-manager/blob/master/docs/development.md

**Comment by fritzmg** [comment]
Right, thank you. Unfortunately I was not able to run the frontend server:
```
Errors:
  45  http://eslint.org/docs/rules/linebreak-style
 @ ./~/babel-loader/lib!./~/vue-loader/lib/selector.js?type=script&index=0!./src/components/views/Account.vue 15:17-48
 @ ./src/components/views/Account.vue
 @ ./~/babel-loader/lib!./~/vue-loader/lib/selector.js?type=script&index=0!./src/components/App.vue
 @ ./src/components/App.vue
 @ ./src/main.js
 @ multi ./build/dev-client ./src/main.js
Child html-webpack-plugin for "index.html":
         Asset     Size  Chunks  Chunk Names
    index.html  1.48 MB       0
    chunk    {0} index.html 542 kB [entry] [rendered]
        [0] ./~/lodash/lodash.js 540 kB {0} [built]
        [1] ./~/html-webpack-plugin/lib/loader.js!./src/index.html 690 bytes {0} [built]
        [2] (webpack)/buildin/global.js 509 bytes {0} [built]
        [3] (webpack)/buildin/module.js 517 bytes {0} [built]
webpack: Failed to compile.
```
Any ideas? That error does not say much at all 😁 

I also had to update the node modules, otherwise node-sass would not be able to run under Windows 10 x64 with the newest Node.js version (11.2.0).

**Comment by aschempp** [comment]
Please try the latest master. Did you install all composer and npm dependencies?

**Comment by fritzmg** [comment]
I have updated the master branch and also ran
```
$ composer install
$ rm package-log.json
$ npm install
```
Now I get a different error:
```
$ npm run serve

> contao-manager@ serve …\…\contao-manager
> vue-cli-service serve

 INFO  Starting development server...
 ERROR  Error: Cannot find module 'webpack'
Error: Cannot find module 'webpack'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:603:15)
    at Function.Module._load (internal/modules/cjs/loader.js:529:25)
    at Module.require (internal/modules/cjs/loader.js:658:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (…\…\contao-manager\node_modules\webpack-dev-server\lib\Server.js:35:17)
    at Module._compile (internal/modules/cjs/loader.js:722:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:733:10)
    at Module.load (internal/modules/cjs/loader.js:620:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:560:12)
    at Function.Module._load (internal/modules/cjs/loader.js:552:3)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! contao-manager@ serve: `vue-cli-service serve`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the contao-manager@ serve script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     …\scoop\persist\nodejs\cache\_logs\2018-11-21T10_36_37_802Z-debug.log
```
To fix this `webpack` needs to be installed globally or locally. So in my case I did
```
npm install -g webpack
npm link webpack
```
However, now I get the following error:
```
Module build failed (from ./node_modules/postcss-loader/src/index.js):
Error: Loading PostCSS Plugin failed: Cannot find module 'autoprefixer'
```
The autoprefixer is installed at this location though:
```
node_modules\@vue\cli-service\node_modules\autoprefixer
```
:/

**Comment by aschempp** [comment]
interesting! I have installed Vue CLI, maybe that is a requirement for the project? http://cli.vuejs.org

**Comment by fritzmg** [comment]
> interesting! I have installed Vue CLI, maybe that is a requirement for the project? http://cli.vuejs.org

Yep, that was it, now it's up and running, finally 😎

**Comment by aschempp** [comment]
implemented in version 1.1.0
